### PR TITLE
Feature commands

### PR DESCRIPTION
**Pull request by dwillmer**
Currently having to use `any` for the commands argument to avoid a circular dependency with phosphide. Otherwise is good to go.

### CODE REVIEW

**Comment by dwillmer** [comment]
...fixing build failure on example now...

**Comment by dwillmer** [comment]
We need to decide what to do here - by keeping the keymap manager in this repo, yet dependent on the command registry (phosphide), we have a circular dependency. As a result i haven't updated the example, as it doesn't make sense to have the example here dependent on phosphide. Possibly move the example to phosphide so we can demonstrate the whole system?

**Comment by sccolbert** [comment]
Before executing the command, it should be checked that it's enabled, and skipped if it's not.

**Comment by sccolbert** [comment]
What do you think about a more specific name for the has/get methods? I can imagine someone wanting to query by sequence as well as command.

**Comment by sccolbert** [comment]
The query methods should probably return arrays since a command can be registered more than once, or take an args object to compare against commandArgs to narrowbthe match.

**Comment by sccolbert** [comment]
Could also just return an array of matched items, and let the user filter. If we do that, we probably want to freeze the items on intake, or hand out shallow copies.

**Comment by dwillmer** [comment]
The command registry `execute` method already checks if it's enabled - https://github.com/phosphorjs/phosphide/blob/ec6cfd885eeb04f54dce439c036e1f84e12801d0/src/commandregistry/plugin.ts#L169

**Comment by dwillmer** [comment]
(ie, we should probably put that logic in a single place)

**Comment by dwillmer** [comment]
I'll update the names to `hasCommand`/`getCommand`

File: src/manager.ts [hunk 1]
```diff
@@ -126,13 +135,46 @@ class KeymapManager {
...
+  }
+
+  /**
+   * Retrieves the sequences registered for the given command.
+   *
+   * @param command - The command of interest.
+   *
+   * @returns The string sequences for the command.
+   */
+  getSequencesForCommand(command: ICommand): string[][] {
```
Comment (dwillmer) [inline src/manager.ts]: Changed method name to `getSequencesForCommand` + return value is an array of sequences.

**Comment by sccolbert** [comment]
closing in favor of keeping handler function (as discussed)

**Pull request closed without merging**